requirements.txt

```diff
@@ -28,7 +28,7 @@ python-memcached==1.59
 raven==6.9.0
 repoze.lru==0.7
 requests==2.19.1
-simplejson==3.16.1
+simplejson==3.16.0
```

Comment:
Why did we need to rollback?

Reply:
https://github.com/simplejson/simplejson/issues/237

Reply:
Yes, as noted in the commit message.